### DISCUSSION TITLE
Objects not showing up

### DISCUSSION THREAD

**Discussion by Tiffanycanny** [CollectionBuilder/collectionbuilder.github.io#125] [Q&A]
I can't figure out why my objects won't show up. I know it has to be something with my metadata and the format. They started as pdf's but they didn't show up. I converted the files to jpgs and then added them under objects and they are still not showing up. I'm not sure what to do.

**Comment by owikle** [comment]
@Tiffanycanny you just need to add file extensions to the values in your filename field. If you add .jpg to all of the filenames all but two will then display on your website. The two that don't display (2206_001 and 2207_001) seem to only have pdf objects related to them. If you want them to show as pdfs, add .pdf as the extension for their filenames and change the value of format to application/pdf. However, both of those pdf files seem to be empty. They won't open on my computer or in the browser. So consider swapping them out for the originals. Hope this helps!

**Reply by Tiffanycanny** [reply]
I added the file extensions but the objects still won't show up on the website.

**Reply by owikle** [reply]
You'll need to do it in your collmeta spreadsheet. That's the one you've configured in your _config.yml file. It looks like you updated a different metadata spreadsheet.

**Comment by Tiffanycanny** [comment]
I got it! Now I just have to figure out the pdfs! Thanks!!

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
